pkg/sql/parser/sql.y

```diff
@@ -4634,7 +4634,7 @@ create_stmt:
 //  ON 'stream_uri'
 //  INTO <TABLE remote_name | TABLES (remote_name, ...) | DATABASE remote_name>
 //  [WITH
-//  < MODE = immediate | transactional > |
+//  < MODE = immediate | validated | transactional > |
```

Comment:
[nit, existing] Can we remove transaction from this comment. I don't think we would even call a future mode transactional since it is too easily misunderstood.